packages/lumx-react/src/components/tabs/Tab.tsx

```diff
@@ -61,23 +63,26 @@ const Tab: React.FC<TabProps> = ({
             return;
         }
 
-        onTabClick?.({ event, index });
+        onChange?.(tabKey as string);
     };
 
     return (
-        <a
+        <button
+            aria-controls={tabKey}
```

Comment:
To be a11y complient, the tabs really should have the `tab` role.

```suggestion
            role="tab"
            aria-controls={tabKey}
```

that way assistive technologies can make the link between the `tablist` parent and the actual tabs.

Also `aria-controls` has to be the id of the `tabpanel` element linked to this tab. (the content). Therefore setting it without having set a corresponding id to it's content might be confusing for the user :/ (And also, it might break our Lighthouse score... which I'm pretty sure someone will check at some point).
I'm not sure how we could make it easier for the user to make the link between the two... 

Also, regarding the tab panels themselves, they should have the following attributes to be complient

```
  role="tabpanel"
  id="someUniqueId"
  aria-labelledBy="theTabId"
```

Maybe we could add a TabPanel component to manage this ? I don't know... Just a though

Reply:
I don't know we could enforce a11y on tab panels since they are not DS component.
About `aria-controls`, I am well aware and that's why I had written the following as possible improvements:
![Capture d’écran de 2020-11-17 09-10-56](https://user-images.githubusercontent.com/2828353/99363359-ebc96200-28b4-11eb-91c7-172be001eb93.png)

We should discuss about the tab panels. Maybe we should create an empty wrapper `TabPanel` including a11y attributes :man_shrugging: What do you think ?

Thanks for the `role`s btw, I wasn't reading correctly the table from https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html (weird format imo)